source/utilities.tex

```diff
@@ -4872,6 +4711,28 @@
 otherwise \tcode{get<$i$>(v) >= get<$i$>(w)} with $i$ being \tcode{v.index()}.
 \end{itemdescr}
 
+\indexlibrarymember{operator<=>}{variant}%
+\begin{itemdecl}
+template<class... Types> requires (ThreeWayComparable<Types> && ...)
+  constexpr common_comparison_category_t<compare_three_way_result_t<Types>...>
+    operator<=>(const variant<Types...>& v, const variant<Types...>& w);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effect
+Equivalent to:
+\begin{codeblock}
+if (v.valueless_by_exception() && w.valueless_by_exception())
+  return strong_ordering::equal;
+if (v.valueless_by_exception()) return strong_ordering::less;
+if (w.valueless_by_exception()) return strong_ordering::greater;
+if (auto c = v.index() <=> w.index(); c != 0) return c;
+return get<i>(v) <=> get<i>(w);
```

Comment:
This `i` should be _`i`_, or `$i$` in latex.

Reply:
Fixed.